css/navigation.css

```diff
@@ -0,0 +1,72 @@
+.navigation-simple {
+  margin: 2rem;
+}
+
+.navigation-simple li a:hover {
+  color: var(--main-logo-color);
+}
+
+.submenu-content {
+  display: none;
+  position: absolute;
+  left: 52%;
```

Comment:
It might be possible that I was wrong here, 
But I think so ,
When you reuse this submenu on another project, it causes alignment issues bz "left : 52%".
52% consider your web screen's left side alignment.
For submenu use -> left:0; for reusable purpose.

Reply:
You are right.

But if i do left:0  then the item goes out of my display container and to the edge of the screen.